src/components/Inputs/__snapshots__/Input.test.js.snap

```diff
@@ -0,0 +1,34 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Is this file necessary? Mabye add it to .gitignore, just asking

Reply:
I mean the folder __snapshots__

Reply:
ok, I'll fix it :)